### PR TITLE
Mat synth tweaks

### PR DESCRIPTION
**Pull request by ghost**
[grammar][tweak]

### CODE REVIEW

File: code/game/objects/items/devices/mat_synth.dm [hunk 1]
```diff
@@ -33,6 +33,29 @@
...
+							 "plasma" = /obj/item/stack/sheet/mineral/plasma,
+							 "uranium" = /obj/item/stack/sheet/mineral/uranium,
+							 "plastic" = /obj/item/stack/sheet/mineral/plastic,
+							 "mythril" = /obj/item/stack/sheet/mineral/mythril,
+							 "bananium" = /obj/item/stack/sheet/mineral/clown,
+							 "phazon" = /obj/item/stack/sheet/mineral/phazon,
+							 "telecrystal" = /obj/item/stack/sheet/mineral/telecrystal,
+							 "sandstone" = /obj/item/stack/sheet/mineral/sandstone,
+							 "bricks" = /obj/item/stack/sheet/mineral/brick)
+
```
Comment (Eneocho) [inline code/game/objects/items/devices/mat_synth.dm:58]:
```suggestion
max_matter = INFINITY
```
This would help with testing.

Comment (ghost) [reply code/game/objects/items/devices/mat_synth.dm:58]: Matter isn't even used at all on the admin ones, so this is pointless
Also there's no max matter var

**Comment by SonixApache** [comment]
neither of those require a :cl: entry

File: code/game/objects/items/devices/mat_synth.dm (hunk 1 above)
Comment (Eneocho) [reply code/game/objects/items/devices/mat_synth.dm:58]: I had meant to post matter = infinity. It doesn't use matter? You learn something new.

**Pull request merged**